cmd/node/main.go

```diff
@@ -239,6 +241,11 @@ VERSION:
 		Usage: "This flag specifies the logger level",
 		Value: "*:" + logger.LogInfo.String(),
 	}
+	//logFile is used when the log output needs to be logged in a file
+	logFile = cli.BoolFlag{
+		Name:  "logFile",
```

Comment:
please rename this.  this does not sound like a bool flag. maybe `save log` and don't use camel case in flags. The rest of the flags in the node should be refactored also. so maybe `save-log` for flag name and `saveLog` for the variable

Reply:
done, both log-related flags. The rest should be done in a separate PR as there might be implications regarding the scripts